recipe/fd_setsize.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/Modules/selectmodule.c b/Modules/selectmodule.c
+--- a/Modules/selectmodule.c
++++ b/Modules/selectmodule.c
+@@ -29,7 +29,7 @@
+    FD_SETSIZE higher before this; e.g., via compiler /D switch.
+ */
+ #if defined(MS_WINDOWS) && !defined(FD_SETSIZE)
+-#define FD_SETSIZE 512
++#define FD_SETSIZE 2048
```

Comment:
Like any arbitrary limit, it just feels _a priori_ sufficient :-) Note there's a slight runtime cost as the fd array is runtime allocated; I think the cost is practically negligible but still.
By default:
```
>>> a, b= socket.socketpair()
>>> %timeit select.select([a], [a], [a])
The slowest run took 12.48 times longer than the fastest. This could mean that a
n intermediate result is being cached.
100000 loops, best of 3: 2.89 µs per loop
```
With this PR:
```
>>> %timeit select.select([a], [a], [a])
The slowest run took 24.39 times longer than the fastest. This could mean that a
n intermediate result is being cached.
100000 loops, best of 3: 3.52 µs per loop
```